### DISCUSSION TITLE
Wallaby.js compatibility with pnp.cjs?

### DISCUSSION THREAD

**Discussion by jbaccarelli-equinix** [yarnpkg/berry#4439] [Q&A]
Hi there, I'm having a million issues trying to run wallaby.js with pnp enabled.
As soon as I configure `nodeLinker: node-modules` everything starts to work again.

I've tried every tutorial and thread online to make this work with the zip files but i can't find a solution.

My last idea would be to somehow tell yarn to install node_modules normally for my local version of the repo, but without modifying the cache or leaving any committable trace behind.
Maybe some command like `yarn install --nodeLinker node_modules` that would not modify the pnp.cjs file or .cache.

Is there something like that? or does anyone have any ideas how to get this going?

Thanks and regards

**Comment by arcanis** [comment]
The `.pnp.cjs` file and `node_modules` are mutually exclusive. If your repository use PnP, I'd suggest to ask Wallaby to ensure they are compatible with it.
